app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command = command.split()
+    with open(command[1], "r") as file_in, open(command[2], "w") as file_out:
```

Comment:
It must do nothing in case the user is trying to copy file to file with the same name.